CrocSwap/ambient-ts-app
# Refactor range button message + InitPool reorder state fxns

- [ ] Refactors logic for range component button message - it's been simplified to a useMemo and is made reusable in the initPool page
- [ ] Reorder so that all useStates are at the top and can be used by any subsequent logic - this should be a standard rather than use states throughout the component. 


To test:
- Creating range in normal tap should still function as normal (message when inputting above your balance etc)
- Pool init now has this kind of messaging when trying to create range positions.
